examples/item_lists.rs

```diff
@@ -0,0 +1,92 @@
+extern crate quire;
+#[macro_use] extern crate serde_derive;
+use quire::{parse_config, Options};
+use quire::validate::{Structure, Scalar, Sequence};
+//e quire::validate::{Structure, Scalar, Sequence, Enum};
+// TODO  Marker_002  Validate against known vehicles
+
+
+#[derive(Deserialize)]
+#[allow(dead_code)]
+struct Journey {
+    name: String,
+    year: String,
+    team: Members,
+    vehicles: Vehicles,
+    locations: Locations,
+}
+
+#[derive(Debug,Deserialize)]
+struct Members(Vec<String>);
+
```

Comment:
Should there be something like
```rust
struct JourneyIter {
   team: Iterator;
   locations: Iterator;
   vehicles: Enumerator:
}
```

Or does `rust-quire` that already  and have I no clue  how to use it? ( Yes,I'm the person that opened #21, Item lists example   `:-/` )